docs/contributing/development/continuous_integration.rst

```diff
@@ -10,270 +10,22 @@ or a change is merged into the *master* branch, a service will
-  variable (alias ``Build.SourcesDirectory``) when fetching a single repository,
-  but after fetching a second repository code is moved automatically to
-  ``Build.Repository.LocalPath/repository-name``.
-
-See the Azure documentation to learn more about `checking out multiple repositories`_.
-
-
-Jobs
-----
-
-You can organize your pipeline into jobs. Every pipeline has at least one job.
-A job is a series of steps that run sequentially as a unit. In other words,
-a job is the smallest unit of work that can be scheduled to run.
-
-
-.. code-block:: yaml
-
-  jobs:
-  - job: myJob
-
-    pool:
-      vmImage: 'ubuntu-latest'
-
-    steps:
-    - bash: echo "Hello world"
-
-Jobs can run in parallel (for example: run the same job on multiple OSes) or
-depend on a previous job.
-
-See the `Azure documentation section on jobs`_ for more information.
-
+Currently, we use GitHub Actions to run all of our pipelines. Making changes to an existing 
+pipeline is as easy as making a pull request. To create a new workflow on GitHub, 
+just create a new YAML file in ``.github/workflows``.
 
 TARDIS Pipelines
 ----------------
 
-Brief description of pipelines already implemented on Azure or GitHub Actions.
-
-
-The default template
-====================
-
-Templates let you define reusable content, logic, and parameters. It functions
-like an include directive in many programming languages (content from one file
-is inserted into another file).
-
-The common set of steps used across most TARDIS pipelines resides in the
-"default" template:
-
-- Force ``set -e`` on all Bash steps.
-- Set TARDIS custom variables.
-- Fetch TARDIS main repository.
-- Fetch TARDIS reference data repository from mirror (optional).
-- Configure Anaconda for Linux and macOS agents.
-- Install Mamba package manager (optional).
-- Install TARDIS environment (optional).
-- Build and install TARDIS (optional).
-
-It was written to make pipelines easier to create and maintain. For example,
-to start a new pipeline use::
-
-  steps:
-    - template: templates/default.yml
-      parameters:
-        useMamba: true
-
-**List of template parameters:**
-
-- ``fetchDepth`` (*int*): the depth of commits to fetch from ``tardis`` repository,
-  default is ``0`` (no limit).
-- ``fetchRefdata`` (*bool*): fetch the ``tardis-refdata`` repository from Azure Repos,
-  default is ``false``.
-- ``refdataRepo`` (*option*): source of the ``tardis-refdata`` repository,
-  options are ``azure`` (default) or ``github``.
-- ``useMamba`` (*bool*): use the ``mamba`` package manager instead of ``conda``,
-  default is ``false``. 
-- ``tardisEnv`` (*bool*): setup the TARDIS environment, default is ``true``.
-
-**List of predefined custom variables:**
-
-- ``tardis.dir`` is equivalent to ``$(Build.SourcesDirectory)/tardis``.
-- ``refdata.dir`` is equivalent to ``$(Build.SourcesDirectory)/tardis-refdata``.
-
-See the `Azure documentation section on templates`_ for more information.
-
```

Comment:
This can be improved/modified according to our current pipeline setup, I think deleting this will remove some nice documentation.

Reply:
I dont think we have anything significant which we can add here, it would be just like copying the github docs.